classrooms/models.py

```
@@ -1,3 +1,83 @@
+            self.slug = slugify(self.title)
+        super().save(*args, **kwargs)
+
+    def get_teachers(self):
+        return self.teachers.all()
+
+
+class ClassroomTeacher(BaseModel):
+    ROLE_CHOICES = [
+        ('teacher', 'Teacher'),
+        ('assistant', 'Teaching Assistant'),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)